### PR TITLE
[OMNI-2298] Add datepicker

### PR DESCRIPTION
**Pull request by bc-ninablanson**
# WHAT
Adding ability to choose datepicker for datetime-local

# WHY
Need datepicker

# TESTING
Locally

### CODE REVIEW

File: src/services/schema-form.js [hunk 1]
```diff
@@ -15,19 +15,26 @@ angular.module('schemaForm').provider('schemaForm',
         return type;
       }
 
+      var titleCase = function(input) {
+        input = input || '';
+        var out = input.split(/(?=[a-z])(?=[A-Z])|(?=[A-Z])(?=[A-Z][a-z])/).join(' ')
```
Comment (rtalvarez) [inline src/services/schema-form.js:20]: what's this sorcery? something with capitalizing stuff?

Comment (bc-ninablanson) [reply src/services/schema-form.js:20]: yea. So now inside selects it wont be `ToysAndGames` but `Toys And Games`
If we ever move this out of JS we can probably be more liberal with our regexes so we can get all of the cases (like the ones w/ underscores, un-capitalize `And`, ect)

Comment (rtalvarez) [reply src/services/schema-form.js:20]: Gotcha

**Comment by rtalvarez** [comment]
👍 🐑

**Pull request merged**